tests/cases/fourslash/signatureHelpRestArgs2.ts

```diff
@@ -0,0 +1,18 @@
+/// <reference path="fourslash.ts" />
+
+// @strict: true
+// @allowJs: true
+// @checkJs: true
+
+// @filename: index.js
+
+//// const promisify = function (thisArg, fnName) {
+////     const fn = thisArg[fnName];
+////     return function () {
+////         return new Promise((resolve) => {
+////             fn.call(thisArg, ...arguments, /*1*/);
```

Comment:
Crash from https://github.com/microsoft/TypeScript/issues/57622#issuecomment-1975331860 happens in completions but it's caused by the same underlying issue. If requested I can add an extra completions-oriented test case too